components/match2/commons/matches_table.lua

```diff
@@ -0,0 +1,340 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:MatchesTable
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local MatchesTable = {}
+
+local Class = require('Module:Class')
+local Countdown = require('Module:Countdown')
+local String = require('Module:StringUtils')
+local HiddenSort = require('Module:HiddenSort')
+local Logic = require('Module:Logic')
+local Table = require('Module:Table')
+local Lua = require('Module:Lua')
+local MatchGroupWorkaround = require('Module:MatchGroup/Workaround')
+local DisplayHelper = Lua.import('Module:MatchGroup/Display/Helper', {requireDevIfEnabled = true})
+
+-- overridable if wikis have custom modules
+MatchesTable.OpponentDisplay = Lua.import('Module:OpponentDisplay', {requireDevIfEnabled = true})
+MatchesTable.Opponent = Lua.import('Module:Opponent', {requireDevIfEnabled = true})
+
+local _UTC = ' <abbr data-tz="+0:00" title="Coordinated Universal Time (UTC)">UTC</abbr>'
+local _TBD = 'TBD'
+local _args
+local _matchHeader
+local _currentID
+
+-- If run in abbreviated roundnames mode
+local _ABBREVIATIONS = {
+	["Upper Bracket"] = "UB",
+	["Lower Bracket"] = "LB",
+}
```

Comment:
Am I misremembering, or don't the messages have support for this (long and short name)

Reply:
ah good point, didn't think about the short stuff

but the _ABBREVIATIONS replacing would still be needed for custom header entries

Reply:
adjusted it to now to use the shorter forms for the default headers, for headers, where it is not set it will do the replacement run (via a now sep. function)